test/tensor_reorder.cpp

```diff
@@ -478,6 +507,7 @@ struct reorder_test : reorder_base
                                 order_3);
 
 #if MIOPEN_BACKEND_OPENCL
+            if(reorder_sol->IsSkippable()){
```

Comment:
line:510 ~ 530 seem duplication of line:477~495?

Reply:
In this test,` dst_dev `memory workspace are controlled by` IsSkippable() `and `GetOutputTensorSize()`. When the reorder could be skipped, e.g. there are nothing changed in tensor memory layout after reorder operation, the workspace of `dst_dev` is assigned with `0 `memory space and we verify the result using the origin tensor on `src_dev`.

Reply:
see blow